cli/azd/pkg/project/scaffold_gen.go

```diff
@@ -270,21 +233,100 @@ func infraSpec(projectConfig *ProjectConfig) (*scaffold.InfraSpec, error) {
 	return &infraSpec, nil
 }
 
-func mapContainerApp(res *ResourceConfig, svcSpec *scaffold.ServiceSpec, infraSpec *scaffold.InfraSpec) error {
-	props := res.Props.(ContainerAppProps)
+func mapUses(infraSpec *scaffold.InfraSpec, projectConfig *ProjectConfig,
```

Comment:
In the new function, it handles all `uses`:
1.  Hosting service uses backing services. Link `api` uses `mysql`.
2. One hosting services uses another hosting service. Like `web` uses `api`.